src/pages/career.astro

```diff
@@ -74,23 +78,73 @@ const { frontmatter, compiledContent } = Astro.props
         <div class="w-full flex justify-center pt-10">
           <div class="w-3/4 grid grid-cols-1 lg:grid-cols-3 align-middle text-center justify-center">
             {
-              posts.map((post) => {
+              roles.map((role) => {
                 return (
                   <>
                   <div class="flex flex-col gap-3 text-center pb-12 lg:pt-0">
-                    <img class="w-24 mx-auto" src={`/icons/${post.frontmatter.icon}`}></img>
-                    <p style="word-spacing:99999px" class="font-bold text-2xl">{post.frontmatter.role}</p>
-                    <a href={`/roles/${post.frontmatter.url}`} class=" text-mcswf-gold font-bold">Learn More</a>
+                    <img class="w-24 mx-auto" src={`/icons/${role.frontmatter.icon}`}></img>
+                    <p style="word-spacing:99999px" class="font-bold text-2xl">{role.frontmatter.role}</p>
+                    <a href={`/roles/${role.frontmatter.url}`} class=" text-mcswf-gold font-bold">Learn More</a>
                   </div>
                   </>
                 )
               })
             }
           </div>
         </div>
+        <div class="w-full flex justify-center">
+          <WhiteButton
+            text="SEE JOB LISTINGS"
+          />
+        </div>
+        <div class="w-full flex justify-center">
+          <div class="w-3/4">
+            <MissionMetrics/>
+          </div>
+        </div>
+      </div>
+      <div class="relative pt-20 z-20 bg-white">
+        <div class="w-full flex flex-col justify-center text-center align-center text-black">
+          <h1 class="text-4xl font-bold pb-8">READY TO JOIN?</h1>
+          <p class="pb-8">Apply before 1. September 2025. (Placeholder)</p>
+        </div>
+        <div class="w-full pb-8">
+          <div class="w-10/12 lg:w-8/12 mx-auto">
+            <hr class="h-[1px] bg-mcswf-pinstripe border-none" />
+          </div>
+        </div>
+      </div>
+      <div class="relative pb-12 z-20 bg-white">
+        <div class="w-full flex flex-col justify-center text-center align-center text-black">
+          <h1 class="text-4xl font-bold pb-8">PREREQUISITES</h1>
+          <div class="flex justify-center">
+            <div class="w-3/4 grid lg:grid-cols-2 gap-12">
+              <div>
+                <h2 class="text-center lg:text-left text-mcswf-pinstripe font-bold">PRE-REQ 1</h2>
+                <p class="text-left">Marines with Primary Military Occupational Specialty 0671 and ranks LCpl through Sgt may apply. <br> *Note: LCpls who are not eligible for promotion to the rank of Cpl on or before 1 July 2024 will not be screened or interviewed for the FY24 cohort.</p>
```

Comment:
We have prerequisite content (now out of date). Switching to hardcoding the information seems like a step back.